app/services/order_cycle_form.rb

```diff
@@ -160,4 +183,41 @@ def removed_schedule_ids
   def new_schedule_ids
     @order_cycle.schedule_ids - existing_schedule_ids
   end
+
+  def can_update_selected_payment_methods?
+    @user.admin? || coordinator? || distributor?
+  end
+
+  def can_update_selected_shipping_methods?
+    @user.admin? || coordinator? || distributor?
+  end
+
```

Comment:
As far as I can see these two methods are doing the same thing, maybe they can be combined : 
```ruby
def can_update_selected_payment_or_shipping_methods?
    @user.admin? || coordinator? || distributor?
end
```

Reply:
Agreed with above ^